.github/AI_REVIEWERS.md

```
@@ -17,16 +19,34 @@ This repo uses two free AI review tools on every PR, plus local Claude Code revi
 
 **Ignoring suggestions**: Reply to the comment explaining why you disagree — CodeRabbit learns from this feedback. Or just resolve the thread.
 
-## Qodo Merge (PR-Agent)
+## Qodo PR-Agent
 
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)